### ISSUE TITLE
OpenFiles context provider does not work when folder is opened over Remote-SSH

### ISSUE THREAD

**Issue by johansigfrids** [continuedev/continue#2451]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: Windwos 11 => Ubuntu 22.04.5 LTS
- Continue: v0.9.214 (pre-release)
- IDE: Visual Studio Code 1.93.1
- Model: Same with GPT-4o and Clause 3.5 Sonnet
- config.json:
  
{
  "models": [
    {
      "title": "OpenAI GPT-4o",
      "provider": "openai",
      "model": "gpt-4o",
      "apiKey": "<REDACTER>",
      "contextLength": 128000,
      "maxTokens": 4095
    },
    {
      "model": "claude-3-5-sonnet-20240620",
      "provider": "anthropic",
      "cacheSystemMessage": true,
      "apiKey": "<REDACTER>",
      "title": "Claude 3.5 Sonnet",
      "contextLength": 128000,
      "maxTokens": 4095
    }
  ],
  "slashCommands": [
    {
      "name": "edit",
      "description": "Edit selected code"
    },
    {
      "name": "comment",
      "description": "Write comments for the selected code"
    },
    {
      "name": "share",
      "description": "Export this session as markdown"
    },
    {
      "name": "cmd",
      "description": "Generate a shell command"
    }
  ],
  "contextProviders": [
    {
      "name": "code",
      "params": {}
    },
    {
      "name": "docs",
      "params": {}
    },
    {
      "name": "diff",
      "params": {}
    },
    {
      "name": "open",
      "params": {
        "onlyPinned": false
      }
    },
    {
      "name": "url"
    },
    {
      "name": "terminal",
      "params": {}
    },
    {
      "name": "problems",
      "params": {}
    },
    {
      "name": "folder",
      "params": {
        "nRetrieve": 150,
        "nFinal": 20,
        "useReranking": true
      }
    },
    {
      "name": "codebase",
      "params": {
        "nRetrieve": 150,
        "nFinal": 20,
        "useReranking": true
      }
    },
    {
      "name": "repo-map"
    },
    {
      "name": "tree"
    }
  ],
  "completionOptions": {
    "maxTokens": 4095
  },
  "tabAutocompleteModel": {
    "title": "Codestral",
    "provider": "mistral",
    "model": "codestral-latest",
    "apiKey": "<REDACTER>",
    "contextLength": 32000
  },
  "tabAutocompleteOptions": {
    "useCopyBuffer": true,
    "useFileSuffix": true,
    "multilineCompletions": "auto",
    "useOtherFiles": true,
    "useCache": true
  },
  "allowAnonymousTelemetry": true,
  "embeddingsProvider": {
    "provider": "openai",
    "model": "voyage-code-2",
    "apiBase": "https://api.voyageai.com/v1/",
    "apiKey": "<REDACTER>"
  },
  "reranker": {
    "name": "voyage",
    "params": {
      "apiKey": "<REDACTER>"
    }
  },
  "docs": []
}
```

### Description

When using the Remote-SSH feature to edit a repo on a Ubuntu machine from my Windows machine, the OpenFiles context provider does not include any of the open files in the prompt. Manually including the same files using Files provider works fine. I have verified this by looking at the output in Output > Continue - LLM Prompt/Completion. When using Files the file shows up in prompt. When opening the file and using OpenFiles the file does not.

I've included the log output for both a prompt using Files and a prompt using OpenFiles.

### To reproduce

1. Use Remote-SSH to open a folder on a remote computer
2. Open a file on the remote computer
3. Use the OpenFiles context provider to ask an AI model something
4. Check the output logs and see that the files contents were not included in the prompt

### Log output

```Shell
==========================================================================
==========================================================================
##### Completion options #####
{
  "contextLength": 128000,
  "maxTokens": 4095,
  "model": "gpt-4o"
}

##### Request options #####
{}

##### Prompt #####
<user>
\home\johan\ai-assist-dm\test.js
function foo() {
  console.log("Hello World")
}
```

test.js What does the function foo log?

==========================================================================
==========================================================================
Completion:

The function `foo` logs the string `"Hello World"` to the console. When the function is called, it will execute the `console.log` statement, displaying the text `"Hello World"` in the console.

==========================================================================
==========================================================================
##### Completion options #####
{
  "contextLength": 128000,
  "maxTokens": 4095,
  "model": "gpt-4o"
}

##### Request options #####
{}

##### Prompt #####
<user>
What does the function foo log?

==========================================================================
==========================================================================
Completion:

To determine what the function `foo` logs, we first need to see the implementation or at least a description of the `foo` function. Please provide the code or description, and I'll help explain what it logs.
```

**Comment by sestinj** [comment]
@johansigfrids it looks like this has been solved by the PR linked above—can you verify that this is working in 0.9.215?

**Comment by johansigfrids** [comment]
Yes this is fixed in v0.9.215 (pre-release) 🎉

**Comment by tobiajung** [comment]
@sestinj 
Hello, when is this fix expected to be in the release version?
I use v0.8.54 and the problem still occurs there. I do not want to switch to pre-release though

**Comment by sestinj** [comment]
@tobiajung We released 0.8.55 on Friday, so this should now be ready

**Comment by tobiajung** [comment]
@sestinj Yes it works, thanks a lot!
